settings.gradle

```diff
@@ -1,8 +1,13 @@
+pluginManagement {
+    plugins {
+        id 'org.jetbrains.kotlin.jvm' version '2.0.0'
```

Comment:
Why do we use kotlin in lw-java?

Reply:
At some point I must have added a Kotlin file by mistake and IDEA must have added this automatically. I removed it now